hypha/apply/stream_forms/forms.py

```diff
@@ -41,8 +47,35 @@ def hidden_fields(self):
 
 
 class PageStreamBaseForm(BaseForm, StreamBaseForm):
-    # Adds page and user reference to the form class
-    pass
+    """ Adds page and user reference to the form class"""
+
+    def clean(self):
```

Comment:
@frjo I checked this piece of code is not getting used by any review forms or determination forms. Both(Review and determination) forms have their own form classes. So, I think we can move forward with this change. What do you think?